documentation/book/proc-kafka-connect-mounting-volumes.adoc

```diff
@@ -0,0 +1,89 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-kafka-connect-external-configuration.adoc
+
+[id='proc-kafka-connect-mounting-volumes-{context}']
+
+= Mounting Secrets as volumes
+
+This procedure shows how to use an {ProductPlatformName} Secret, mount it as volume into Kafka Connect, and use it to configure Kafka Connect connector.
+
+.Prerequisites
+
+* A running Cluster Operator.
+
+.Procedure
+
+. Create a secret containing a properties file with the configuration options we want to use in our connector configuration.
```

Comment:
_Please reword:_

Create a secret containing a properties file that defines the configuration options for your connector configuration.